openbb_platform/core/openbb_core/app/static/package_builder.py

```diff
@@ -1079,6 +1079,10 @@ def get_param_info(parameter: Parameter) -> Tuple[str, str]:
 
         docstring = summary.strip("\n").replace("\n    ", f"\n{create_indent(2)}")
         docstring += "\n\n"
+        if Env().LLM_MODE:
+            if len(docstring) > 1024:
+                docstring = docstring[:1021] + "..."
+            return docstring
```

Comment:
This also doesn't seem to work correctly, eg. calling `obb.equity.price.quote.__doc__` returns the full docstring.

Reply:
Did you rebuild?